.github/workflows/check_lint.yml

```diff
@@ -0,0 +1,35 @@
+name: lint check
+
+# When the workflow will be triggered
```

Comment:
_nit:_ I'm in favor of consistent commenting. We could either add this comment and the following (`# Step 1: Check out the repository` etc.) to `.github/workflows/check_code_vulnerabilities.yml`, or delete these comments altogether from `.github/workflows/check_lint.yml` and `.github/workflows/check_unit_tests.yml`, if we think the code reads for itself (for example, the `name` arg seems to be communicating the same information).

Reply:
I'll clean them up a bit and only leave in comments where I think its useful.  FWIW, chatgpt auto generated some of these initial workflows, hence the duplicative nature between names and comments.  I should have taken a deeper look before committing.